.github/dependabot.yml

```diff
@@ -3,8 +3,16 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: monthly
```

Comment:
monthly seems a bit too long, how about weekly? :)

Reply:
I guess for me, monthly seemed like a decent interval to try and keep packages uptodate; weekly feels too noisy offhand, ie I'd just end up ignoring them (though I don't mind trying it first!)

Reply:
I'll put it to weekly for now :)